.github/workflows/release.yml

```diff
@@ -7,9 +7,48 @@ on:
       - "v*"
   pull_request:
     branches: [main]
+  workflow_dispatch:
+    inputs:
+      wash_version:
+        description: "Version of wash, without the `v`, to release to chocolatey"
+        required: true
+        default: "0.18.0"
 
 jobs:
+  update-release:
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Prepare Formula
+        env:
+          WASMCLOUD_URL: https://github.com/wasmCloud/wash/releases/download/v{{ inputs.wash_version }}/wash.exe
+        run: |
+          choco_template=$(cat ./template/chocolateyinstall.txt)
+          nuspec_template=$(cat ./template/wash.txt)
+
+          wasmcloud_url=$WASMCLOUD_URL
+          wasmcloud_sha=$(curl -sL $WASMCLOUD_URL | shasum -a 256 | cut -d ' ' -f 1 | tr '[:lower:]' '[:upper:]')
+          wasmcloud_version={{ inputs.wash_version }}
+
+          choco=$(echo "$choco_template" | sed "s|WASMCLOUD_URL|$wasmcloud_url|")
+          choco=$(echo "$choco" | sed "s|WASMCLOUD_SHA|$wasmcloud_sha|")

```

Comment:
Why not pipe out to sed a second time in the previous line?

Reply:
Ask ChatGPT 😬

Reply:
I'm gonna merge just to test it since I can't run the workflow yet, but happy to enhance this a little bit :)